app/forms.py

```diff
@@ -6,10 +6,13 @@
 
 class LoginForm(FlaskForm):
     username = StringField('Username', validators=[DataRequired()])
-    password = PasswordField('Password', validators=[DataRequired(), Length(min=8, max=30),  Regexp(r'[A-Za-z0-9]+',message='Password must contain Uppercase, lowercase and a number')])
+    password = StringField('Password', validators=[DataRequired()])
```

Comment:
why changed it to StringField? wont this not have the hide effects for password?

Reply:
True, ill change it tomorrow and repush. Was just getting errors using password fix so used this for testing.